manubot/manubot.py

```diff
@@ -0,0 +1,317 @@
+import argparse
+import collections
+import datetime
+import json
+import logging
+import os
+import pathlib
+import re
+import sys
+import textwrap
+
+import errorhandler
+import jinja2
+import pandas
+import requests
+import requests_cache
+import yaml
+
+from manubot.manuscript import (
+    get_citation_strings,
+    get_manuscript_stats,
+    get_text,
+    replace_citations_strings_with_ids,
+)
+from manubot.citations import (
+    citation_to_citeproc,
+    get_citation_id,
+    is_valid_citation_string,
+    standardize_citation,
+)
+
+
+def parse_arguments():
+    """
+    Read and process command line arguments.
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--content-directory', type=pathlib.Path, required=True)
+    parser.add_argument('--output-directory', type=pathlib.Path, required=True)
+    parser.add_argument('--template-variables-path',
```

Comment:
Good point, will add an example manuscript under `tests` for testing `--template-variables-path`